test/ntp_client_proposal_test.rb

```diff
@@ -0,0 +1,169 @@
+require_relative "test_helper"
+
+require_relative "../src/clients/ntp-client_proposal.rb"
```

Comment:
this is a bit problematic, as it execute client program. In this case it just write error to log, but it can change in future ( e.g. many client raise exception if parameter missing or start UI). That is reason why code in src/clients have also zero code coverage.

Reply:
Ops!

Sorry, I forgot that. Probably you already told me in a previous PR.

So, shall I to discard this test? Or mock the call to `log.error` in this case and keep the test?

What do you prefer or think is better?

Thank you.

Reply:
keep it as it is for now